WallPaperDev/WallPaperDev/Views/SearchImages Views/SearchImageCell.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  SearchImagesCell.swift
+//  WallPaperDev
+//
+//  Created by Jamar Gibbs on 12/3/19.
+//  Copyright © 2019 Stephen Ouyang. All rights reserved.
+//
+
+import UIKit
+
+class SearchImagesCell : UICollectionViewCell {
+  let identifier = "searchCell"
+  
+  // Using layers to hold uninteractable images
+  lazy var photoLayer = CALayer()
+  lazy var maskLayer = CAShapeLayer()
+  lazy var borderLayer = CAShapeLayer()
+  
+  var cellImage: UIImage? {
+    didSet {
+      photoLayer.contents = cellImage?.cgImage
+    }
+  }
+  
+  func getImage(_ image: UIImage?) {
+    photoLayer.backgroundColor = UIColor.red.cgColor
```

Comment:
Theme

Reply:
This was just a placeholder color to show the cell. I'll remove it soon.